src/index.js

```diff
@@ -611,30 +611,27 @@ f.delay = function (ms) {
 };
 
 // Exported variables
-exports.bone = bone; // this likely needs to be platform and be detected
-for (var x in f) {
-    exports[x] = f[x];
-}
+module.exports.bone = bone; // this likely needs to be platform and be detected
 for (var x in functions) {
-    exports[x] = functions[x];
+    f[x] = functions[x];
```

Comment:
What does this impact regarding load speed and memory? Can we optimize that? BoneScript already takes an excessive time to load.

Reply:
As far as i know this change might not make much change to the load speed and memory usage , removing some unused exports after analyzing is sure to improve the performance of the library.